deps/compose.template

```diff
@@ -44,9 +44,7 @@ services:
       bootstrapper_l1:
         condition: service_completed_successfully
     volumes:
-    {%- if IS_NOT_CI %}
-      - ./data/madara:/usr/share/madara/data
-    {%- endif %}
+      - /tmp/madara:/usr/share/madara/data
```

Comment:
why madara-db for appchain was moved to /tmp? It might be inconvenient to make a fresh start and we'll lose the db after a restart from the host?